apps/vanilla/src/pages/exercise1/style/components.css

```
@@ -173,7 +173,7 @@
   border-color: var(--on-surface-color);
 }
 
-.select-wrapper select:not(:checked) {
+.select-wrapper select:invalid {
```

You have `.select` class

```suggestion
.select-wrapper .select:invalid {
```